typhon/cloudmask/aster.py

```diff
@@ -58,6 +58,15 @@ def __init__(self, filename):
         self.scenecenter = SceneCenter(
             *self._convert_metastr(meta['SCENECENTER'], dtype=tuple)
         )
+        CornerCoordinates = namedtuple('CornerCoordinates', 
+                                       ['LOWERLEFT', 'LOWERRIGHT', 
+                                        'UPPERRIGHT', 'UPPERLEFT'])
```

Comment:
I would use lowercased variable names for the "Python world". But I am fine with sticking to the ASTER naming scheme.